src/blog/2019-census/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: The 2019 Reactiflux Census
+author: vcarl
+date: '2020-07-07'
+description: 'We recently ran a user survey with the aim of better understanding just who is using Reactiflux (or, at least, those who felt comfortable answering a voluntary survey with demographic questions). The results are in.'
+---
+
+# Reactiflux census 2019
+
+Towards the end of 2019, we ran a user survey with the aim of better understanding just who is using Reactiflux (or, at least, who felt comfortable answering a voluntary survey with demographic questions). The results are in.
+
+The results were actually in a while back, but we got caught up with implementing a number of changes to moderation processes and tools, and ultimately delayed this post by a number of months. Sorry! The upside is that we'll also be adding an official Code of Conduct shortly.
```

Comment:
Personally I'd prefer to avoid mentioning the CoC until it's ready to be enforced, as we'll probably get a lot of negative feedback about it. After it's published, we could edit this to include a follow-up link.
```suggestion
The results were actually in a while back, but we got caught up with implementing a number of changes to moderation processes and tools, and ultimately delayed this post by a number of months. Sorry!
```